bin/interpreter.sh

```diff
@@ -72,77 +72,60 @@ fi
 
 # set spark related env variables
 if [[ "${INTERPRETER_ID}" == "spark" ]]; then
-  # add Hadoop jars into classpath
-  if [[ -n "${HADOOP_HOME}" ]]; then
-    # Apache
-    addEachJarInDir "${HADOOP_HOME}/share"
-
-    # CDH
-    addJarInDir "${HADOOP_HOME}"
-    addJarInDir "${HADOOP_HOME}/lib"
-  fi
+  if [[ -n "${SPARK_HOME}" ]]; then
+    export SPARK_SUBMIT="${SPARK_HOME}/bin/spark-submit"
+    SPARK_APP_JAR="$(ls ${ZEPPELIN_HOME}/interpreter/spark/zeppelin-spark*.jar)"
+    # This will evantually passes SPARK_APP_JAR to classpath of SparkIMain
+    ZEPPELIN_CLASSPATH=${SPARK_APP_JAR}
```

Comment:
@weipuz Thanks for digging it ! 
I noticed that issue too. ( I set `SPARK_HOME` and can not get spark log file. )
I also changed  `ZEPPELIN_CLASSPATH=${SPARK_APP_JAR}` to `ZEPPELIN_CLASSPATH+=${SPARK_APP_JAR}` as you said. Then finally I can get my `zeppelin-interpreter-spark-***.log` file. As long as it's not an intended implementation, I think we need to fix this.